src/services/session.service.ts

```diff
@@ -0,0 +1,38 @@
+import Session from "../session/session";
+import { IPromiseToFileSession } from "session/types";
+import { saveSession } from "./redis.service";
+import * as keys from "../session/keys";
+
+/**
+ * Creates a new ptf session and adds a company to the context.
+ * This will override any ptf session that is already in redis.
+ * @param chSession
+ * @param companyNumber
+ */
+const createPromiseToFileSession = async (
+    chSession: Session, companyNumber: string): Promise<IPromiseToFileSession> => {
+
+    const existing = chSession.data[keys.PTF_SESSION];
+    if (existing) {
+        return existing;
+    }
+
+    const ptfSession: IPromiseToFileSession = {
+        company_number_in_context: companyNumber,
+        ptf_requests: [],
```

Comment:
Do we need this yet? Probably not, since this PR is all about retrieving company details based on the number.

Reply:
Removed